marconi-sidechain-experimental/src/Marconi/Sidechain/Experimental/Api/JsonRpc/Endpoint/EpochActiveStakePoolDelegation.hs

```diff
@@ -1,12 +1,76 @@
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
```suggestion
```

Reply:
Applied -- Thanks for the speedy review.